config/sidekiq.yml

```diff
@@ -2,4 +2,5 @@
 :max_retries: <%= ENV.fetch("SIDEKIQ_MAX_RETRIES", 2) %>
 :queues:
   - shrine
+  - mailers
```

Comment:
Do we need this?

Reply:
Acionmailer dropped the messages on the mailers queue. I needed to setup sidekiq to read it in